applications/services/desktop/scenes/desktop_scene_pin_input.c

```diff
@@ -121,8 +133,19 @@ bool desktop_scene_pin_input_on_event(void* context, SceneManagerEvent event) {
         case DesktopPinInputEventResetWrongPinLabel:
             desktop_scene_locked_light_red(false);
             desktop_view_pin_input_set_label_primary(desktop->pin_input_view, 0, 0, NULL);
-            desktop_view_pin_input_set_label_secondary(
-                desktop->pin_input_view, 44, 25, "Enter PIN:");
+            uint32_t attempts = furi_hal_rtc_get_pin_fails();
+            state->enter_pin_string = furi_string_alloc();
+            furi_string_set_str(state->enter_pin_string, "Enter PIN (Fails: ");
```

Comment:
This can be cleaned up a bit by doing something like:

```
furi_string_set_str(state->enter_pin_string, "Enter PIN (Fails: %lu):");
furi_string_printf(state->enter_pin_string, attempts);
```

And it makes the full string easier to read!